lib/iphone-simulator-xcode-7.ts

```diff
@@ -57,7 +57,7 @@ export class XCode7Simulator extends IPhoneSimulatorNameGetter implements ISimul
 			let launchResult = this.simctl.launch(device.id, applicationIdentifier).wait();
 
 			if (options.logging) {
-				this.printDeviceLog(device.id, launchResult);
+				this.getDeviceLogProcess(device.id, launchResult);
```

Comment:
In fact the `run` method is called when the ios-sim-portable is called as a separate process. Sooo I've broken printing the logs here. I'll fix this ;)